tonic/models/vic/vic2netcdf.py

```diff
@@ -424,11 +417,9 @@ def nc_fields(self, fields, y_x_dims, precision):
                                                           zlib=False)
 
                 if 'units' in field:
-                    self.fields[name].long_name = name.encode()
-                    self.fields[name].coordinates = 'lon lat'.encode()
+                    self.fields[name].long_name = name
+                    self.fields[name].coordinates = 'lon lat'
                     for key, val in field.items():
-                        if isinstance(val, str):
-                            val = val.encode()
```

Comment:
You'll need to undo all the changes to the string encoding that came in here.  You can do that manually or by using something like source tree.

The reason this happened is that you copied in an old version of this file (that wasn't in git) into the current develop branch so you've just undone all the changes we've recently made to the file, that git has recorded.  Does that make sense?